src/test/scala/uk/gov/hmrc/perftests/example/TrackingConsentFrontendRequests.scala

```diff
@@ -40,8 +40,20 @@ object TrackingConsentFrontendRequests extends ServicesConfiguration {
       .check(status.is(200))
   }
 
+  val getLiveHmrcFrontendVersion: String = {
+    val url = "https://staging.tax.service.gov.uk/help/cookie-details"
```

Comment:
could we use baseUrlFor("tracking-consent-frontend") I think this path is also to help-frontend not tracking-consent-frontend - could just use baseUrlFor("tracking-consent-frontend") and it will be a 404 but should still work I think

Reply:
could use this, from looking at how we setup the requests
```
$baseUrl/tracking-consent/cookie-settings
```